eos_validator.py

```diff
@@ -0,0 +1,51 @@
+
+# Check the TOKEN amount
+def check_balance(node_host="http://localhost:8888", snapshot_csv="./snapshot.csv", eos_total=1000000000):
+    with open(snapshot_csv, 'r') as fp:
+        account_onchain_total = 0
+        for onepersion in fp.readline()
```

Comment:
This is ok for checking that all the buying got their EOS, but is missing checking that no other account was created/staked.

Reply:
yes, there is no method/api which can get all created accounts in the chain, so I just make sure the account from snapshot is correct. And the check of 
`if abs(eos_total - (account_onchain_total + eosio_onchain_balance)) > 0.0001:`
will make sure even there are extra account, but there will no token transfer to them.

Reply:
ah cool